files/package-plugin.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+packageDirectory="zip_dir"
+
+ZIPNAME="K10rPlugin"
+if [ ! "$1" == "" ]; then
+    ZIPNAME="$1"
+fi
+
+mkdir -p "${packageDirectory}"
+
+for path in `git ls-files | grep -v .gitignore`; do
+	file=$(basename "$path")
+	dir=$(dirname "$path")
+
+	mkdir -p "${packageDirectory}/${dir}"
+	cp "${path}" "${packageDirectory}/${path}"
+done
+
+cd "${packageDirectory}"
+
+composer install -qn --no-dev --ignore-platform-reqs
+
+if [ !  -f .sw-zip-blacklist ]; then
+    cp -r /tmp/.sw-zip-blacklist.dist .sw-zip-blacklist
+fi
+
+while read $b; do
+    rm -rf $b
```

Comment:
Kannst du `$b` noch in Quotes setzen wg. etwaiger Leerzeichen/Sonderzeichen?

Reply:
Und `$b` in sowas wie `$blacklistedFile` ändern?

Reply:
Ist `-rf` notwendig? Würde maximal `-r` machen wg. Verzeichnissen.

Reply:
Vielleicht ist es auch sinnvoller `--exclude @.sw-zip-blacklist .sw-zip-blacklist` zu nutzen.

Reply:
Verwende jetzt die Änderungen aus dem untersten Kommentar